### PR TITLE
Siax - property filtering

### PR DESCRIPTION
**Pull request by jn1118**
Git pull and then yarn build, everything works fine locally.

### CODE REVIEW

**Comment by lmy0125** [comment]
Which page/file did you import your component? Did you remove the file that you used for development?

**Comment by jn1118** [comment]
> Which page/file did you import your component? Did you remove the file that you used for development?

Yea, I removed it because I thought we would change the layout eventually after integrating with Aaditya's part. It was originally in index.js. Should I add it back?

File: src/components/propertyFilter.js [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+  const [sizeMin, setSizeMin] = useState("");
+  const [sizeMax, setSizeMax] = useState("");
+  const [distance, setDistance] = useState("");
+  const [parkingChecked, setParkingChecked] = React.useState(false);
+  const [laundryChecked, setLaundryChecked] = React.useState(false);
+  const [poolChecked, setPoolChecked] = React.useState(false);
+  const [gymChecked, setGymChecked] = React.useState(false);
+  const [acChecked, setAcChecked] = React.useState(false);
+  const [petChecked, setPetChecked] = React.useState(false);
+  const [smokingChecked, setSmokingChecked] = React.useState(false);
```
Comment (lmy0125) [inline src/components/propertyFilter.js:31]:
```suggestion
  const [parkingChecked, setParkingChecked] = useState(false);
  const [laundryChecked, setLaundryChecked] = useState(false);
  const [poolChecked, setPoolChecked] = useState(false);
  const [gymChecked, setGymChecked] = useState(false);
  const [acChecked, setAcChecked] = useState(false);
  const [petChecked, setPetChecked] = useState(false);
  const [smokingChecked, setSmokingChecked] = useState(false);
```

**Review by lmy0125 — APPROVED** [review]
LGTM

**Pull request merged**